### ISSUE TITLE
Indepth tutorial requested

### ISSUE THREAD

**Issue by Kapowpenguin1** [Microsoft/malmo#640]
I am a super noob dealing with this kinda stuff, so I'd appreciate an idiot-proof guide on how to do this, preferably as a video. Just for reference, I am on High Sierra, and have gotten as far as working with .bashrc, and I'm not entirely sure if I did it correctly.

**Comment by Phantomb** [comment]
1. Have you managed to install the dependencies and start "launchclient"?
2. Have you had a look at http://microsoft.github.io/malmo/0.30.0/Python_Examples/Tutorial.pdf ? This is a pretty good starting point if you take the time to do every 'exercise'

**Comment by Kapowpenguin1** [comment]
I have installed the dependencies, is launchclient a thing you do in terminal? Also, will give the link a try.

**Comment by Phantomb** [comment]
Yeah `launchclient.bat` for Windows or `launchclient.sh` for Mac/Linux is the file that starts the Minecraft server. So you open one terminal, navigate to where that file is located, and run that file. Then you open a second terminal from which you can start an agent in your preferred language (the 'default' or most widely used language is python, there are also many examples available for python). These things are also explained at the start of the tutorial, so I really advise you to walk through that and play around with the commands / concepts introduced there.

If you still have specific or broad questions afterwards, feel free to open an issue, or join the gitter: gitter.im/Microsoft/malmo where you can chat with others about the malmo project.
